### DISCUSSION TITLE
How do i get for vs code?

### DISCUSSION THREAD

**Discussion by kailando** [arcticicestudio/nord#257] [Help & Support]
...

**Comment by Bad3r** [comment]
search in the marketplace for "Nord"

**Comment by 1chooo** [comment]
@kailando 

You can find in the Extension and remember to search "Nord". 

Cheers!

<img width="1280" alt="Screenshot 2024-03-16 at 12 18 33 AM" src="https://github.com/nordtheme/nord/assets/94162591/6f5338bb-f365-4fe0-ba05-a1c2d15d356c">
